public/app/features/dashboard/components/ShareModal/ShareExport.tsx

```diff
@@ -115,6 +112,10 @@ export class ShareExport extends PureComponent<Props, State> {
     });
     const time = new Date().getTime();
     saveAs(blob, `${dash.title}-${time}.json`);
+    reportInteraction(shareAnalyticsEventNames.sharingActionClicked, {
```

Comment:
@polibb you may want to implement a function `reportSharingActionClicked` higher-order function that will wrap `reportInteraction` calls rather than duplicating the `reportInteraction(shareAnalyticsEventNames.sharingActionClicked, ...)` all over the place :) Similarily to https://github.com/grafana/grafana/blob/main/public/app/features/alerting/unified/Analytics.ts#L94